src/main/resources/db/changesets/20181203_studytable.yaml

```diff
@@ -0,0 +1,208 @@
+databaseChangeLog:
+  - changeSet:
+      id: study_init
+      author: jhert
+      changes:
+        - createTable:
+            tableName: study
+            columns:
+              - column:
+                  name: id
+                  type: ${uuid_type}
+                  defaultValueComputed: ${uuid_function}
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(1024)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: description
+                  type: varchar(2048)
+                  constraints:
+                    nullable: true
+
+        - createTable:
+            tableName: study_table
+            columns:
+              - column:
+                  name: id
+                  type: ${uuid_type}
+                  defaultValueComputed: ${uuid_function}
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: study_id
+                  type: ${uuid_type}
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk_study_table_study
+                    references: study(id)
+              - column:
+                  name: name
+                  type: varchar(128)
```

Comment:
Size mismatch with the referenced column. Could we make a type, like you did with uuid, for `identifier` so we could adjust the size in one place and keep the definitions in sync?